pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -433,6 +501,16 @@ func updateNodesStatus(c client.Client, instance *computenodev1alpha1.ComputeNod
 						break
 					}
 				}
+				// trigger node drain post tasks (new pod)
+				err = triggerNodeDrain(c, node.Name, instance, false, openstackClientAdmin, openstackClient)
+				if err != nil {
+					return err
+				}
+				// if post draining has finished (pod completed) remove the pod
```

Comment:
need to check this as this can fail if the status already changed. maybe we should just do a regular cleanup of finished pre/post drain pods.